modules/ruxhal/src/time.rs

```diff
@@ -8,7 +8,7 @@
  */
 
 //! Time-related operations.
-
+use core::sync::atomic::{AtomicU64, Ordering};
```

Comment:
this line should also be put under `#[cfg(all(feature = "rtc", target_arch = "x86_64"))]`.